Code/Java/Fibonacci_Series.java

```diff
@@ -0,0 +1,129 @@
+/*The Fibonacci sequence is a series where the next term is the sum of pervious two terms. The first two terms of the Fibonacci sequence is 0 followed by 1.
+The Fibonacci sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21....
+*/
+/**
```

Comment:
Heyy, since working in open source, we donot generally mention our names as the contributor of every code. Pl delete this comment. :)